code/modules/mafia/map_pieces.dm

```diff
@@ -0,0 +1,71 @@
+/obj/effect/landmark/mafia_game_area //locations where mafia will be loaded by the datum
+	name = "Mafia Area Spawn"
+	var/game_id = "mafia"
+
+/obj/effect/landmark/mafia
+	name = "Mafia Player Spawn"
+	var/game_id = "mafia"
+
+/obj/effect/landmark/mafia/town_center
+	name = "Mafia Town Center"
+
+//for ghosts/admins
+/obj/mafia_game_board
+	name = "Mafia Game Board"
+	icon = 'icons/obj/mafia.dmi'
+	icon_state = "board"
+	var/game_id = "mafia"
+
+/obj/mafia_game_board/attack_ghost(mob/user)
+	. = ..()
+	var/datum/mafia_controller/MF = GLOB.mafia_game
+	if(!MF)
+		MF = create_mafia_game()
```

Comment:
how did this compile?

Reply:
why would it not?

Reply:
nevermind, github hid the diff for controller.dm so I thought it trying to call was an undefined proc